articles/connectivity/conn-faq.md

```diff
@@ -27,30 +27,28 @@ As our Assured OFFICIAL cloud platform is PSN accredited, you can also connect t
 
 ## Can I connect to my solution using SSL VPN?
 
-Our Assured OFFICAL and Elevated OFFICAL security domains both support SSL VPN (configured from within the edge gateway), but we don't recommend it other than for occasional remote administration purposes by a small number of administrators. If you need more extensive use of SSL VPN, we recommend you deploy a dedicated VPN appliance, such as OpenVPN.
+Our Assured OFFICAL and Elevated OFFICAL security domains both support SSL VPN (configured from within the edge gateway), but we do not recommend it other than for occasional remote administration purposes by a small number of administrators. If you need more extensive use of SSL VPN, we recommend you deploy a dedicated VPN appliance, such as OpenVPN.
```

Comment:
are we making these changes?

Reply:
Happy to use non-contracted form here for emphasis